docs/contrib/06_new_project.md

```diff
@@ -0,0 +1,40 @@
+# Adding a project to pySDC (and automatic testing)
+
+When adding a new project to pySDC, it needs to be tested.
+In order to run those tests as fast as possible, a few steps are needed, that are given below.
+
+## Add a directory in 'pySDC/projects'
+
+First, create a new directory in `pySDC/projects` with the name of your project.
+The code of the new project should go into that newly created directory.
+
+## Add an environment-file
+
+In addition to the codde of your projects, an environment-file is needed.
+This includes the dependencies that are required for your project to run.
+The file needs to be named `env_update.yml` and needs to follow the structure shown below:
```

Comment:
`env_update.yml` is outdated

Reply:
Name of the file is updated in the next commit.